src/components/families/family.model.js

```diff
@@ -348,6 +348,7 @@ Family.add( 'Permissions', {
 		familyStoryCreated: { type: Types.Boolean, label: 'family story created', default: false, initial: true },
 		videoCreated: { type: Types.Boolean, label: 'video created', default: false, initial: true }
 	},
+		visitingResource: { type: Types.Boolean, label: 'Visiting Resource', default: false, initial: true },
```

Comment:
The `visitingResource` field should be added under of the `familyServices` collection (directly following `videoCreated`) for consistency with the rest of the Family Services fields.